Makefile

```diff
@@ -141,6 +149,27 @@ kind-load-test-artifacts: $(KIND) ## Load the e2e testdata container images into
 	$(KIND) load docker-image localhost/testdata/bundles/plain-v0/plain:v0.1.0 --name $(KIND_CLUSTER_NAME)
 	$(KIND) load docker-image localhost/testdata/catalogs/test-catalog:e2e --name $(KIND_CLUSTER_NAME)
 
+.PHONY: deploy-local-registry
+deploy-local-registry: ## Deploy local docker registry
+	$(CONTAINER_RUNTIME) run -d -p 5001:5000 --restart=always --name local-registry registry:2
+
+.PHONY: stop-local-registry
+stop-local-registry: ## Stop local registry
+	$(CONTAINER_RUNTIME) container stop local-registry
+
+.PHONY: remove-local-registry
+remove-local-registry: ## Remove local registry
+	$(CONTAINER_RUNTIME) container rm -v local-registry
```

Comment:
Having these as separate targets means two things:
They aren't cleaned up when an error occurs, which is good if you want to look at the status of an error, but bad if you want to run the tests again. You might want to force stop in the deploy-local-registry. Or possibly combine these into a single "cleanup" target to make it easier to clean up after failure,

Reply:
Added `cleanup-local-registry` which combines both the targets mentioned above. Need to look into force stop in deploy-local-registry if that is required to be addressed.